src/service/run-api-service.js

```diff
@@ -12,6 +12,7 @@
  *
  * * `account`: Epicenter account id (**Team ID** for team projects, **User ID** for personal projects).
  * * `project`: Epicenter project id.
+ * * `id`: (Optional) The run id of an existing run. If there is a specific run you'd like to work with, you can include it.
```

Comment:
`id` is just an alias for `filter`, think that's worth calling out (I know it's mentioned in the config section below but i'm assuming this appears above that?)